src/utils/user.ts

```diff
@@ -66,8 +66,17 @@ export const isAccessibilityTeam = (
   );
 };
 
-export const isBasicUser = (groups: Array<String> = []) => {
-  return groups.includes(BASIC_USER_PROD) || groups.length === 0;
+export const isBasicUser = (groups: Array<String> = [], flags: Flags) => {
+  if (groups.includes(BASIC_USER_PROD)) {
```

Comment:
We could do this as one big  `return (...) || (...) || (...)` if you like, but I see how writing it in this way makes it more clear.

Reply:
@Igarfinkle yeah... three branches with all of the negatives was a bit too much for my brain to manage. :-D